lms/product/plugin/plugin.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+
+from lms.product.plugin.grouping_service import GroupingServicePlugin
+
+
+@dataclass
+class PluginConfig:
+    """A collection of plugin class definitions."""
+
+    # These also provide the default implementations
+    grouping_service: type = GroupingServicePlugin
```

Comment:
This is the same class as before. You'll also notice that the products haven't changed at all through this, and the interface is exactly the same.